### DISCUSSION TITLE
Value lists and field "active"

### DISCUSSION THREAD

**Discussion by urskaufmann** [teksi/Home#106] [General]
In TWW-project, all fields with value relation and vl-value-lists are configured with filter expression "active".
In the Docu is now in Layer explanation/Value lists writen: `These value lists are defined in the VSA-datamodel. Do not change.`
But for really using the filter expression, somebody has to change in field active.
Several questions: 
- where to mention this in the docu? User Guide (which chapter?), Admin Guide? 
- Shoud/can the TWW-User change the field?
- If somebody change this field somewhere in this vl in some records: how can this be transferred to another database (in case of an update or in case you want to have the some configuration in database B as you have in database A)?

**Comment by cymed** [comment]
only the roles txx_manager and txx_sysadmin have the rights to alter a value list.

**Reply by urskaufmann** [reply]
so we mention this possibility only in the Admin Guide and not in the User Guide.
Third question still open (sysadmin have to lock them self?)

**Reply by cymed** [reply]
So far, yes. There are standardised cases in the AG-xx extenstion where we have pre-defined scripts for it

**Comment by ponceta** [comment]
To answer the third question, it is not part of INTERLIS Export but database configuration.

That's part of the upcoming project to take care of the upgrade mechanics.
